guides/source/routing.md

```diff
@@ -24,7 +24,7 @@ The Rails router recognizes URLs and dispatches them to a controller's action, o
 
 When your Rails application receives an incoming request for:
 
-```
+```plain
```

Comment:
What will be the difference between `plain` and unmarked code fences?

Reply:
They will be similar. At first I tried using the `http` lexer but these require the protocol as well for the requests.
```http
"GET /photos/7 HTTP/1.1"
```
I'll remove these changes..